templates/vc10filters.mpd

```diff
@@ -7,7 +7,6 @@
 <%if(!compares(grouped_source_file, default_group))%>
     <Filter Include="<%grouped_source_file%>">
       <UniqueIdentifier>{<%new_guid(grouped_source_file)%>}</UniqueIdentifier>
-      <Extensions><%foreach(ext, extensions(source_files))%><%ext%><%fornotlast(";")%><%endfor%></Extensions>
```

Comment:
Why remove this?

Reply:
with this line files in sub folder will not added to the solution correctly, on the other hand, the group fashion is different  from vc9 does which show the sub folder in the solution view as it's disk position, I haven't fix this